downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -84,6 +84,126 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 - Introduced GraphQL server-side caching support
 
+#### Introduction of the `data.jsondata` package
+
+- The [`data.jsondata`](https://lib.ballerina.io/ballerina/data.jsondata/latest/) package has been introduced to provide a set of APIs for JSON data conversions, data projection, and JSON navigation.
+
+    ##### Data Projection
+
+    ```ballerina
+    import ballerina/data.jsondata;
+    import ballerina/io;
+
+    // Define a closed record type to capture the required fields from the JSON content.
+    type Book record {|
+        string name;
+        string author;
+    |};
+
+    public function main() returns error? {
+        json jsonContent = {
+            name: "Clean Code",
+            author: "Robert C. Martin",
+            year: 2008,
+            publisher: "Prentice Hall"
+        };
+        // Based on the expected type, it includes only the `name` and `author` fields in the converted value.
+        Book book = check jsondata:parseAsType(jsonContent);
+        io:println(book);
+
+        string jsonStr = string `
+        {
+            "name": "The Pragmatic Programmer",
+            "author": "Andrew Hunt, David Thomas",
+            "year": 1999,
+            "publisher": "Addison-Wesley"
+        }`;
+        Book book2 = check jsondata:parseString(jsonStr);
+        io:println(book2);
+    }
+    ```
+
+    ##### Json Navigation
```

Comment:
```suggestion
    ##### JSON Navigation
```

Add some information. There's nothing (other than comments) that convey JSONPath support here.